Library/Homebrew/manpages/brew.1.md.erb

```diff
@@ -138,6 +138,15 @@ Note that environment variables must have a value set to be detected. For exampl
 
     *Default:* `~/Library/Caches/Homebrew`.
 
+  * `HOMEBREW_CC`:
```

Comment:
I'd probably rather we left this undocumented as it's very likely to break things if you set it in your environment.